extensions/2.0/Khronos/KHR_avatar_mesh_annotation_rendering/README.md

```diff
@@ -0,0 +1,96 @@
+# KHR_avatar_mesh_annotation_rendering
+
+## Contributors
+
+- Ken Jakubzak, Meta
+- Hideaki Eguchi / VirtualCast, Inc.
+- K. S. Ernest (iFire) Lee, Individual Contributor / https://github.com/fire
+- Shinnosuke Iwaki / VirtualCast, Inc.
+- 0b5vr / pixiv Inc.
+- Leonard Daly, Individual Contributor
+- Nick Burkard, Meta
+
+## Status
+
+**Draft** – This extension is not yet ratified by the Khronos Group and is subject to change.
+
+## Dependencies
+
+Written against the glTF 2.0 specification.  
+Dependent on: `KHR_avatar`
+
+## Overview
+
+The `KHR_avatar_mesh_annotation_rendering` extension provides structured metadata for avatar mesh primitives related to **visibility and render-time behavior**. It complements the core glTF material and LOD systems by exposing semantic visibility rules such as “first-person only”, “always visible”, etc.
+
+This extension is intended to support use cases such as:
+
+- Hiding certain parts of an avatar (like head or hair) in first-person mode
+- Marking components that must always be rendered (e.g., eyes)
+- Toggling cosmetic parts based on camera mode or gameplay state
+- Managing specialized render passes (shadows, reflections, etc.)
+
+## Use Cases
+
+- First-person camera hiding for head/hair geometry
+- Consistent behavior for accessories across render modes
+- Semantic control of pass inclusion (e.g., do not render in reflection pass)
+- Optimization by excluding off-screen or nested mesh content
+
+## Extension Schema
+
+This extension is defined per `mesh.primitive`, allowing fine-grained render behavior control.
+
+```json
+{
+  "meshes": [
+    {
+      "primitives": [
+        {
+          "extensions": {
+            "KHR_avatar_mesh_annotation_rendering": {
+              "renderVisibility": "firstPersonOnly",
+              "renderTags": ["eyewear", "forceRender"]
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+### Properties
+
+| Property         | Type        | Description                                                                 |
+|------------------|-------------|-----------------------------------------------------------------------------|
+| `renderVisibility` | string    | Controls camera-based visibility. Enum: `"always"` | `"firstPersonOnly"` | `"thirdPersonOnly"` | `"never"` |
```

Comment:
I think it would reduce confusion to explicitly state whether visibility propagates to child nodes under a node that references this mesh. Since it can be set per primitive, I expect it does not propagate tho.